developer-portal/logo.md

```diff
@@ -1,17 +1,37 @@
 # Change your logo
 
-## Add your logo file
+## Add your logo
 
-Add your logo file.
-If you want to overwrite the existing logo, it's at `images/logo.png`.
+To display your logo in the navbar, overwrite the existing image at `./images/logo.png`.
```

Comment:
It might be helpful to mention if any other image formats can be used, as this implies only PNG is supported.

Reply:
From the slack chat - it seems like SVG and JPEG are the other compatible formats. Is that correct?

Reply:
SVG is supported, but I'm not 100% sure about JPG - would have to test it.